styleguide/components/41-organisms/programme/_programme.scss

```diff
@@ -0,0 +1,194 @@
+.programme {
+  width: calc(100% + 2 * #{$gutter-width});
+  margin-right: -$gutter-width;
+  margin-left: -$gutter-width;
+
+  > .inner-box {
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+
+    &::before {
+      @include spot-image('calendar');
+
+      @include tablet {
+        right: 0;
+        left: 0;
+      }
+    }
+  }
+
+  ///
+  /// Programme slot.
+  ///
+  &-slot {
+    max-width: $bp-max-content;
+    margin: 0 auto;
+
+    &-title {
+      margin: 0 auto 1.6rem;
+      font-size: 1.3rem;
+      text-align: center;
+    }
+
+    > dd {
+      max-width: none;
+      padding: 0;
+    }
+
+    & + & {
+      margin-top: 2.8rem;
+    }
+  }
+
+  ///
+  /// Programme teaser.
+  ///
+  &-teaser {
+    @include theme('background-color', 'color-secondary');
+    display: flex;
+    position: relative;
+    flex-direction: column;
+
+    // Children
+    &__top {
+      position: relative;
+      order: 1;
+    }
+
+    &__bottom {
+      order: 2;
+      padding: .8rem;
+    }
+
+    &-overlay-link {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      text-indent: 200%;
+      white-space: nowrap;
+      overflow: hidden;
+      z-index: 1;
+
+      @at-root {
+        a#{&}:hover,
+        a#{&}:focus {
+          background-color: transparent;
+        }
+      }
+    }
+
+    &-subtitle {
+      display: block;
+      font-size: .8rem;
+    }
+
+    .read-more {
+      padding-top: 0;
+      padding-bottom: 0;
+    }
+  }
+
+  ///
+  /// Programme detail
+  ///
+  &-detail {
+    display: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+
+    z-index: 200;
+    overflow-y: auto;
+
+    // States
+    &.visible {
+      display: block;
+
+      + .programme-detail-overlay {
+        display: block;
+      }
+    }
+
+    // Children
+    &-overlay {
+      @include theme('background-color', 'color-primary--darken-3', 'checkbox-with-filter-overlay-background-color');
+      display: none;
+
+      position: fixed;
+      top: 0;
+      left: 0;
+      width: 100vw;
+      height: 100vh;
+      opacity: .7;
+      z-index: 199;
+    }
+
+    .modal-actions {
+      margin-top: -.4rem;
+      margin-right: -.4rem;
+      text-align: right;
+
+      @include tablet {
+        position: absolute;
+        top: 1.2rem;
+        right: 1.2rem;
+      }
+    }
+
+    &-inner {
+      position: relative;
+      min-height: 100vh;
+      padding: 1rem;
+      background-color: color('white');
```

Comment:
Shouldn't this be themified? :)

Reply:
Don't agree. White is white. This corresponds to the page background and that's also not themifiable.